package.json

```diff
@@ -19,7 +19,9 @@
     "access": "public"
   },
   "license": "(Apache-2.0 AND (Apache-2.0 OR OFL-1.1 OR GPL-2.0-only WITH Font-exception-2.0))",
-  "dependencies": {},
+  "dependencies": {
```

Comment:
Node is 'platform'-like dependency. It's defined in `engined` property.

It should be rather

```
{ "engines" : { "node" : "^10.15.3" } }
```
See https://stackoverflow.com/questions/29349684/how-can-i-specify-the-required-node-js-version-in-packages-json

@haraldf, can you confirm or am i completly wrong ...

Reply:
I think you are right, I have updated it